.github/workflows/zz_generated.update_chart.yaml

```diff
@@ -81,9 +81,9 @@ jobs:
           ref: ${{ needs.gather_facts.outputs.branch }}
       - name: Install vendir
         run: |
-          wget https://github.com/vmware-tanzu/carvel-vendir/releases/download/v0.32.2/vendir-linux-amd64 -O /tmp/vendir
+          wget https://github.com/vmware-tanzu/carvel-vendir/releases/download/v0.32.0/vendir-linux-amd64 -O /tmp/vendir
```

Comment:
Why back to .0?

Reply:
Oh yeah that was my bad because I modified the update-chart PR.

I already merged this on devctl so this will be now managed by it.